src/SweetAlert/functions.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+// test comment
+
 if (!function_exists('alert')) {
```

Comment:
This might be changed too: 
```suggestion
if (!function_exists('alert_v2')) {
```